engine/message/db.go

```diff
@@ -569,11 +575,37 @@ func (db *DB) _SendMessages(ctx context.Context, send SendFunc, status StatusFun
 		return errors.Wrap(err, "clear disabled status updates")
 	}
 
-	_, err = tx.Stmt(db.failDisabledCM).ExecContext(execCtx)
+	// processes disabled CMs and writes to alert log if disabled
+	cms, err := tx.Stmt(db.failDisabledCM).QueryContext(execCtx)
```

Comment:
name this `rows` instead of `cms` as a covention, then add a `defer rows.Close()` after the err check.

Reply:
^ the defer is so that we always close the rows, even if we `return err` before we finish reading.